### PR TITLE
Adds drinking functionality to chemical bottles.

### LINKED ISSUE #10614: FEATURE REQUEST: DRINKING FROM BOTTLES
Someone should code drinking from bottles. Also, having bottles break when thrown/over people's head like booze bottles. My potions are too strong for you, traveler. :100:

### PR DESCRIPTION
**Pull request by ghost**
Fixes #10614 

As mentioned above, this basically copypastes the drinking mechanic from beer glasses and such, and applies it to those chemical bottles you can get from chem-masters.

However, this kinda sacrificed being able to cap the bottle, so I've also converted capping to a general verb that beakers and such use, so you can just right click the bottle and press 'toggle seal' from the verb list.

Additionally, the amount you consume from a chem bottle is dependent on its transfer amount, so you can potentially just down it all in one. However, attempting to apply it to another person simply splashes them like before, so you can't just run around force feeding people 30u hyperzine/chloral bottles.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -54,6 +54,44 @@
...
+
+	else if(!src.reagents.total_volume || !src)
+		to_chat(user, "<span class='warning'>\The [src] is empty.<span>")
+		return 0
+
+	else
+		imbibe(user)
+		return 0
+
+/obj/item/weapon/reagent_containers/glass/bottle/proc/imbibe(mob/user) //Drink the liquid within
```
Comment (ComicIronic) [inline code/modules/reagents/reagent_containers/glass/bottle.dm:71]: Rather than copying this from drinks line for line, move it onto reagent_containers.

**Comment by unid15** [comment]
Idea: capping the bottle is done by attack_self(), clicking on a mob on help intent (including yourself) feeds the bottle's contents to the mob, clicking on a mob on any other intent splashes the bottle's contents on the mob

File: code/modules/reagents/reagent_containers/glass.dm [hunk 2]
```diff
@@ -69,6 +69,13 @@
 
 /obj/item/weapon/reagent_containers/glass/attack_self()
 	..()
+	toggle_open()
+
+/obj/item/weapon/reagent_containers/glass/verb/toggle_open()
+	set name = "Toggle seal"
+	set category = "Object"
+	set src in range(0)
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/glass.dm:77]: more verbs, fun

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 3]
```diff
@@ -54,6 +54,44 @@
...
+
+
+	to_chat(user, "<span  class='notice'>You swallow a gulp of \the [src].</span>")
+	playsound(user.loc,'sound/items/drink.ogg', rand(10,50), 1)
+
+	if(isrobot(user))
+		reagents.remove_any(amount_per_transfer_from_this)
+		return 1
+	if(reagents.total_volume)
+		if (ishuman(user))
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: why can only humans drink what about monkeys

File: code/modules/reagents/reagent_containers/glass.dm (hunk 2 above)
Comment (ghost) [inline code/modules/reagents/reagent_containers/glass.dm:77]: Lemme guess, verbs are a no-no?

**Comment by Shadowmech88** [comment]
I don't like the idea of capping being changed to drinking. You'll have people drinking polyacid and shit by mistake for weeks.

I don't mind allowing people to drink from bottles and beakers, but I think it should be done by literally any other method than this.

**Comment by Probe1** [comment]
I can't wait to try and put a lid on a beaker and accidentally drink polyacid.  Edit:  I should have read all the comments first.  What @Shadowmech88 said.

**Comment by ghost** [comment]
Alrighty, I'll rewrite

**Comment by Shadowmech88** [comment]
You don't have to make an entirely new PR, you know. You can push any new changes to this same branch and they'll show up in this PR.

**Comment by ghost** [comment]
Gotta rewrite it entirely, might as well make a new PR

File: code/modules/reagents/reagent_containers/glass/bottle.dm (hunk 1 above)
Comment (ghost) [inline code/modules/reagents/reagent_containers/glass/bottle.dm:71]: Problem with this is that it was modified severely from regular drinking, as imbide on drink uses several variables within, such as lit, gulp_size, etc.

**Comment by ghost** [comment]
5 months, but reapproaching now

**Pull request closed without merging**